packages/ckeditor5-ui/src/editorui/editorui.ts

```diff
@@ -299,6 +305,25 @@ export default abstract class EditorUI extends /* #__PURE__ */ ObservableMixin()
 		this._focusableToolbarDefinitions.push( { toolbarView, options } );
 	}
 
+	/**
+	 * Stores default location config for given menu bar item.
+	 */
+	public addMenuBarItem(
+		config: MenuBarConfigAddedItem | MenuBarConfigAddedGroup | MenuBarConfigAddedMenu
+	): void {
+		this._extraMenuBarItems.push( config );
+	}
+
+	/**
+	 * Returns all default location configs for menu bar items.
+	 *
+	 * @internal
+	 */
+	public getCustomMenuBarItemsLocations(): Array<MenuBarConfigAddedItem | MenuBarConfigAddedGroup | MenuBarConfigAddedMenu> {
```

Comment:
```suggestion
	public getExtraMenuBarItems(): Array<MenuBarConfigAddedItem | MenuBarConfigAddedGroup | MenuBarConfigAddedMenu> {
```

WDYT?

Reply:
This method will be removed anyway, so I don't care about the name.